src/main/java/io/specto/hoverfly/junit/core/Hoverfly.java

```diff
@@ -364,6 +379,31 @@ public void verify(RequestMatcherBuilder requestMatcher, VerificationCriteria cr
         verifyRequest(requestMatcher.build(), criteria);
     }
 
+    public void assertThatNoDiffIsReported(boolean shouldClean) {
```

Comment:
Can we change the variable name to something like `shouldResetDiff`, to be more explicit?

Reply:
renamed in 1b7e8ec